sherlock/resources/data.json

```diff
@@ -2507,6 +2507,15 @@
     "username_claimed": "green",
     "username_unclaimed": "noonewouldeverusethis7"
   },
+  "tiktok": {

```

Comment:
This name is a little misleading because we are not checking through the official TikTok site. I think it would be better if you changed it to **TikTok Online Viewer** instead.

Reply:
Ok

Reply:
Done